package.json

```diff
@@ -13,28 +13,22 @@
   "dependencies": {
     "@aeternity/aepp-sdk": "^12.0.0",
     "core-js": "^3.24.0",
-    "file-loader": "^6.2.0",
     "vue": "^3.2.37",
     "vue-router": "^4.1.2"
   },
   "devDependencies": {
+    "@babel/core": "^7.18.9",
     "@babel/eslint-parser": "^7.18.9",
     "@vue/cli-plugin-babel": "~5.0.8",
     "@vue/cli-plugin-e2e-cypress": "~5.0.8",
     "@vue/cli-plugin-eslint": "~5.0.8",
     "@vue/cli-plugin-router": "~5.0.8",
     "@vue/cli-plugin-unit-jest": "~5.0.8",
     "@vue/cli-service": "~5.0.8",
-    "@vue/compiler-sfc": "^3.2.37",
-    "@vue/eslint-config-standard": "^8.0.0",
-    "@vue/eslint-plugin": "^4.2.0",
```

Comment:
This package was causing instalation issue, it is 5 years old, was used before @vue/cli